hindi/randeep_hooda/randeep.md

```diff
@@ -0,0 +1,3 @@
+Randeep Hooda (pronounced [r?d?i?p hu??a?]; born 20 August 1976) is an Indian film actor who appears in Hindi-language films. Hooda began acting in school productions at a young age. After completing his studies in Melbourne, Australia, he returned to India and began modelling and acting in theatre.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.